src/main/java/sparkmicroblog/Main.java

```diff
@@ -0,0 +1,52 @@
+package sparkmicroblog;
+
+import spark.ModelAndView;
+import spark.Spark;
+import spark.template.mustache.MustacheTemplateEngine;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+/**
+ * Created by darionmoore on 12/14/16.
+ */
+public class Main {
+    static User userName;
```

Comment:
This is a bad variable name.  It is not a user name it is the user object.  Be careful to make good names for your variables.